src/extension/appLauncher.ts

```diff
@@ -220,6 +224,39 @@ export class AppLauncher {
                                 this.logger.info(localize("PrewarmingBundleCache", "Prewarming bundle cache. This may take a while ..."));
                                 return mobilePlatform.prewarmBundleCache();
                             })
+                            .then(() => {
+                                if (launchArgs.platform === "android" && mobilePlatform instanceof AndroidPlatform) {
+                                    return mobilePlatform.startEmulatorIfNotRun(launchArgs.target);
+                                }
+                                else return undefined;
+                            })
+                            .then((emulator: any) => {
+                                if (emulator) {
+                                    if (emulator.emulatorName) {
```

Comment:
If we use interface type for `startEmulator` function we will not need additional checks here